types/highland/index.d.ts

```diff
@@ -1009,6 +1033,8 @@ declare namespace Highland {
 		 */
 		latest(): Stream<R>;
 	}
+
+	type MappingHint = number | string[] | Function;
```

Comment:
Can you use a more specific function type here? Potentially even just `((...args: any[]) => any)`